spec.html

```diff
@@ -13137,10 +13146,10 @@ <h1>ECMAScript Function Objects</h1>
             [[ScriptOrModule]]
           </td>
           <td>
-            a Script Record or a Module Record
+            a Script Record, a Module Record, *null* or ~hidden~.
```

Comment:
i don't think this can be null (or appear to be null in the case of ~hidden~?). See for example https://github.com/tc39/ecma262/pull/1670.

and so this leads me to my question, why not just create a new script record for indirect evaluators?

Reply:
It can be null when you define a function inside an inline event handler in HTML, since in that case GetActiveScriptOrModule() can be null.

> and so this leads me to my question, why not just create a new script record for indirect evaluators?

That would work too, but it has the following challenges:
- we probably need a new host hook to let hosts define the [[HostDefined]] slot of those new script records created within ECMA-262
- it means that `eval("import('foo')"); eval("import('foo')")` could load two different modules, because the cache is per-realm/script/module.